sympy/vector/coordsysrect.py

```diff
@@ -1,20 +1,55 @@
+import types
+from sympy.simplify import simplify
 from sympy.core.basic import Basic
 from sympy.vector.scalar import BaseScalar
-from sympy import eye, trigsimp, ImmutableMatrix as Matrix, Symbol
+from sympy import (eye, trigsimp, ImmutableMatrix as Matrix, Symbol,
+                   sqrt, Lambda, sin, cos, sinh, cosh, S)
 from sympy.core.compatibility import string_types, range
 from sympy.core.cache import cacheit
 from sympy.vector.orienters import (Orienter, AxisOrienter, BodyOrienter,
                                     SpaceOrienter, QuaternionOrienter)
-import sympy.vector
+from sympy.utilities.exceptions import SymPyDeprecationWarning
+
+
+SymPyDeprecationWarning(
+    feature="CoordSysCartesian",
+    useinstead="CoordSystem3D",
+    deprecated_since_version="0.7.6"
+).warn()
```

Comment:
What is the proper way to warn the user about using a deprecated class?  Right now it is outside the class, should it be inside so the *bin/doctests* can run?